src/days/day03.rs

```diff
@@ -42,18 +39,17 @@ impl Map {
     }
 }
 
-type Position = (u32, u32);
-type Slope = Position;
-
-fn apply_slope(slope: Slope, mut position: Position) -> Position {
-    position.0 += slope.0;
-    position.1 += slope.1;
+struct Position(u32, u32);
```

Comment:
Having a Slope type is now pointless, since we can't use it as a constructor, but it doesn't really matter.